upd (#6)
DevExpress-Examples/xaf-how-to-use-data-from-security-in-criterion: Readme.md

## Implementation Details

> Refer to the following help topic to learn how to create such a criterion operator: [Custom Function Criteria Operators](https://docs.devexpress.com/eXpressAppFramework/113480/filtering/in-list-view/custom-function-criteria-operators)

Reviewer: ```suggestion
Refer to the following help topic to learn how to create such a criterion operator: [Custom Function Criteria Operators](https://docs.devexpress.com/eXpressAppFramework/113480/filtering/in-list-view/custom-function-criteria-operators).
```

> Refer to the following help topic to learn how to create such a criterion operator: [Custom Function Criteria Operators](https://docs.devexpress.com/eXpressAppFramework/113480/filtering/in-list-view/custom-function-criteria-operators).

Reviewer 2: ```suggestion
Refer to the following help topic for instructions on how to create these criterion operators: [Custom Function Criteria Operators](https://docs.devexpress.com/eXpressAppFramework/113480/filtering/in-list-view/custom-function-criteria-operators).
```
